configure.ac

```diff
@@ -227,6 +227,20 @@ AM_COND_IF([ENABLE_MANPAGES], [
 	AS_IF([test "x$RST2MAN" = "x"], [AC_MSG_ERROR([[--enable-manpages requires rst2man]])])
 ])
 
+AC_ARG_WITH([bash-completion-dir],
+	AS_HELP_STRING([--with-bash-completion-dir[=PATH]],
+	[Install the bash auto-completion script in this directory. @<:@default=yes@:>@]),
```

Comment:
Agreed - this code relies on functions from the bash-completion project, and so I have used bash-specific extensions since bash-completion also uses them. I think it is possible to make portable completions usable by any readline based shell, but would have to use external tools such as grep/sed/awk where POSIX standard constructs are not sufficient. As I personally do not use any other shells, I do not feel qualified to extend this beyond bash. So I'll make the default "no".